changelog/unreleased/pull-3048

```diff
@@ -1,9 +1,16 @@
-Enhancement: Check now checks index when reading packs
+Enhancement: Check has now more checks for consistency of index and pack files
 
 Restic used to only verfiy the pack file content when calling `check --read-data`
```

Comment:
Strictly not related, but there's a typo here: verify

Reply:
Also, are we missing an "or" at the end of this line?

Reply:
is both fixed.